examples/flows/chat/chat-with-pdf/tests/chat_with_pdf_test.py

```diff
@@ -70,15 +71,18 @@ def test_bulk_run_valid_mapping(self):
         details = self.pf.get_details(run)
         self.assertEqual(details.shape[0], 3)
 
-    # def test_bulk_run_mapping_missing_one_column(self):
-    #     # in this case, run won't be created.
-    #     with self.assertRaises(ValidationException):
-    #         self.create_chat_run(
-    #             column_mapping={
-    #                 "question": "${data.question}",
-    #                 "pdf_url": "${data.pdf_url}",
-    #             }
-    #         )
+    def test_bulk_run_mapping_missing_one_column(self):
+        # in this case, run won't be created.
+        data_path = os.path.join(
+            self.flow_path, "data/invalid-data-missing-column.jsonl"
+        )
+        with self.assertRaises(ValidationException):
+            self.create_chat_run(
+                column_mapping={
+                    "question": "${data.question}",
+                },
+                data=data_path
```

Comment:
what's is the original behavior when `data` is not provided?

Reply:
it will use a default data.